docs/web_advanced.rst

```diff
@@ -896,20 +896,26 @@ background tasks could be registered as an :attr:`Application.on_startup`
 signal handler or :attr:`Application.cleanup_ctx` as shown in the example
 below::
 
-
-  async def listen_to_redis(app):
-      try:
-          sub = await aioredis.create_redis(('localhost', 6379))
-          ch, *_ = await sub.subscribe('news')
-          async for msg in ch.iter(encoding='utf-8'):
-              # Forward message to all connected websockets:
-              for ws in app[websockets]:
-                  ws.send_str('{}: {}'.format(ch.name, msg))
-      except asyncio.CancelledError:
-          pass
-      finally:
```

Comment:
Would it not be a good idea to keep the `finally`? In case there are any errors.
I guess an even better approach would be if the API supported `async with pubsub:`.

Reply:
Applied `async with client.pubsub() as pubsub:` instead.